scipy/stats/_distn_infrastructure.py

```diff
@@ -38,11 +37,9 @@
 
 from ._constants import _XMAX
 
-if PY2:
-    instancemethod = types.MethodType
-else:
-    def instancemethod(func, obj, cls):
-        return types.MethodType(func, obj)
+
+def instancemethod(func, obj, cls):
+    return types.MethodType(func, obj)
```

Comment:
Would it not make more sense to remove the `instancemethod` shim completely?

Reply:
What would the change be?

Reply:
done.